bindings.go

```diff
@@ -461,10 +461,31 @@ func NewQuery(pattern []byte, lang *Language) (*Query, error) {
 
 	return q, nil
 }
+
 func deleteQuery(q *Query) {
 	C.ts_query_delete(q.c)
 }
 
+// FindAll returns all query captures for a node
+func (q *Query) FindAll(n *Node) []*Node {
```

Comment:
Since the captures are named and there may be more than one, I propose to either return `[][]*Node` or `[]map[string]*Node` (key is the capture name).

Alternatively, it may make sense to rename the method to somehow hint that it assumes only one capture, and fail if there are more than one.

Reply:
Thanks. Yes, original idea was to use this method for only one capture.

But it looks like `map[string][]*Node` doesn't make usage more difficult but solves the problem with many captures.

Or do I miss something and it should be `[]map[string]*Node` as you proposed?

Reply:
Yeah, I think it should be `[]map[string]*Node` because it first locates a match and inside each one it it may have multiple named captures.

Reply:
What is the use case for it?
If capture has the same name in different matches they should be treated the same otherwise why would they have the same name?

Reply:
Look at following queries:
```go
query = []string{
		`((import_statement name: (dotted_name (identifier)) @name))`,
		`((import_statement name: (aliased_import name: (dotted_name (identifier)) @name)))`,
		`((import_from_statement module_name: (dotted_name (identifier)) @name))`,
		`((import_from_statement module_name: (relative_import (import_prefix)) @name))`,
	}
```
it's a slice of queries to extract python imports. We can test it one by one (for each query), but we can also combine them and make it as a one big query. But the point is that every independent statement has one _capture_ `name`. So I think `map[string][]*Node` can be a better approach.

Reply:
@kuba-- For this particular use case it's useful to have `map[string][]*Node`.

But if you consider another your use case which is to extract function calls, it contains two captures: `(... @name ... @args)`. For that case it's useful to have `[]map[string]*Node`, e.g. a list of function calls, and then the name+args named captures.

Reply:
but what does it mean, e.g. `[1]["name"]` ?

Reply:
The function name in the second function call

Reply:
for me the use cases are:
- people reused names and they know so they can just iterate.
- people use unique names so they can just go to lookup table and get the slice (if there are more with the same name).

So for me the most natural way (for both cases) is to get nodes by `["name"]`
or function names by `["fn"]` or arguments by `["arg"]`
but for the second case the i-th function name should match the i'th argument

Reply:
Both arguments make sense to me.

I also see that this API would break use cases when the order of captures is important.
Maybe callback API proposed by Kuba will be better.

I will close this PR for now. Thank you.